source/codegen/metadata/nixnetsocket/functions_addon.py

```diff
@@ -4,3 +4,8 @@
 functions_validation_suppressions = {
     name: {"parameters": {"mem": ["ARRAY_PARAMETER_NEEDS_SIZE"]}} for name in ["Recv", "RecvFrom"]
 }
+
+# Void* param output size is allocated differently.
```

Comment:
Allocated with `supports_standard_output_allocation`? Would that be more helpful?

Reply:
Thanks, that previous comment was lacking. Including `supports_standard_output_allocation` here tells why the size requirement can be suppressed.